Makefile.dep

```diff
@@ -564,6 +564,14 @@ ifneq (,$(filter random,$(USEMODULE)))
         USEMODULE += prng_tinymt32
     endif
 
+    ifneq (,$(filter prng_fortuna,$(USEMODULE)))
+        USEMODULE += fortuna
+        USEMODULE += hashes
+        USEMODULE += crypto
+        USEMODULE += xtimer
+        CFLAGS += -DCRYPTO_AES 
```

Comment:
I think the `CFLAGS` is ugly here, but it's the only way to tell the `crypto` module to include AES. It's WIP after all :-)